Owncloud iOs Client/AppDelegate.m

```diff
@@ -58,6 +58,7 @@
 
 
 #define k_server_with_chunking 4.5 
+#define k_DB_version_12 12
```

Comment:
This is not necessary on the Appdelegate now it is on the InitializeDatabase class.